src/utils/browser.ts

```diff
@@ -14,3 +14,8 @@ export const isChromiumBased = (): boolean => !!window.chrome
 export const isAllowedBrowser = () => isFirefox() || isChromiumBased()
 
 export const isBrowserOutdated = !('ResizeObserver' in window) || !('IntersectionObserver' in window)
+
+export const openInNewTab = (url: string, local?: boolean) => {
+  const origin = window.location.origin
+  window.open(`${local ? origin : ''}${url}`, '_blank')?.focus()
+}
```

Comment:
since you've added that utility fn it might make sense to use that here:
https://github.com/Joystream/atlas/blob/dev/src/components/VideoPreview.tsx#L90

not in the scope though

Reply:
done